File: 3731/solution.java
class Solution {
    public List<Integer> findMissingElements(int[] nums) {
        Set<Integer> vis = new HashSet<>();
        int minValue = Integer.MAX_VALUE;
        int maxValue = Integer.MIN_VALUE;
        for(int i: nums){
            minValue = Math.min(minValue, i);
            maxValue = Math.max(maxValue, i);
            vis.add(i);
        }
        List<Integer> res = new ArrayList<>();
        for(int i=minValue;i<maxValue;i++){
            if(!vis.contains(i))
                res.add(i);
        }
        return res;
    }
}

File: 3731/solution.py
class Solution:
    def findMissingElements(self, nums: List[int]) -> List[int]:
        maxValue = float('-inf')
        minValue = float('inf')
        vis = set()
        for i in nums:
            maxValue = max(maxValue, i)
            minValue = min(minValue, i)
            vis.add(i)
        
        res = []
        for i in range(minValue,maxValue):
            if not i in vis:
                res.append(i)
        return res
